cmake/external/abseil-cpp.cmake

```diff
@@ -21,8 +21,8 @@ FetchContent_Declare(
     abseil_cpp
     PREFIX "${CMAKE_CURRENT_BINARY_DIR}/abseil-cpp"
```

Comment:
>"${CMAKE_CURRENT_BINARY_DIR}/abseil-cpp" [](http://example.com/codeflow?start=11&length=40)

I wonder if we could make it uniform for all fetch_declare. Some sort of external, they are all in the same place as it was before and can be automatically found. Perhaps, we could make it so that we do not need modify `Directory.Build.Props.in` and other C# and NuGet building things that may lead to pipeline changes.